documentation/identifiers.md

```diff
@@ -1,70 +1,80 @@
 
-We use simple text identifiers in 360Giving. As well as using these identifiers in your published data, you could also tell your grantees their prefixed identifier, and encourage them to include it in any documents they publish about their funded projects, or to include it as a 'machine tag' (or hashtag) when uploading photos and videos to social media sites. 
+For organisation identifiers, if you can, use an officially recognised identifier for the organisation, following the [organisation identifier](organisation-identifier) guidance below. If you can't do this right away, you can also use your internal identifiers for grantees along with your assigned 360G prefix. 
 
-> For example, some of the development of 360Giving was funded by grant '360G-indigotrust:IND233'. If you [search the web for that identifier](https://www.google.co.uk/search?q=360G-indigotrust%3AIND233), you will find this site. If there were project reports published online about this grant, or photos and videos, you might be able to discover those as well. If you just searched for the **internal identifer** [IND233](https://www.google.co.uk/search?q=IND233) you would have to comb through details of all sorts of other things identified as IND233 before finding any information about the grant.
 
-You can also make use of your 360Giving identifiers in internal reports and documents, as this will make it easier to digitally link these up with the data in your 360Giving data files in future.
+```eval_rst
+.. _register-prefix:
+```
 
-## Register a prefix
+## Get your prefix
 
-To register a prefix see the [publisher guidance](http://www.threesixtygiving.org/data/publish-data/).
+To register a prefix see the [publisher guidance](http://www.threesixtygiving.org/standard/register/).
 
 All registered prefixes should start with 360G unless you have been advised otherwise by the support team. 
 
+```eval_rst
+.. _grant-identifier:
+```
+
 ## Grant Identifier
 
 To create your grant identifiers:
 
-1. Make sure you have registered for 360Giving prefix;
-2. Look for an existing internal identifier given to your grants (This might be a sequential number assigned to each grant at the point of application, or a combination of the 'funding scheme' identifier and a sequential number for the grant. The important thing is that the identifier should be unique inside your organisation, so adding the prefix will make it unique across the whole world.);
-2. Add this onto the end of your 360Giving prefix;
+1. Make sure you have asked for a 360Giving prefix;
+2. Look for an existing internal identifier given to your grants (e.g. a sequential number assigned to each grant at the point of application, or a combination of the 'funding scheme' identifier and a number for the grant. The important thing is that the identifier should be unique inside your organisation, so adding the prefix will make it unique across the whole world.);
+3. Add this onto the end of your 360Giving prefix;
```

Comment:
as it's a prefix - makes sense to order sentence this way
3.	Add your 360Giving prefix in front of your internal identifier;